grin.toml

```diff
@@ -222,11 +222,17 @@ NUM_THREADS = 1
 #[mining.miner_plugin_config.device_parameters.0]
 #NUM_THREADS = 1
 
-#CUDA Miner (Included here for integration only, Not ready for use)
+#CUDA Miner
+#
+# Note! 4+ GB GPU cards only!
+#   If you have 6GB GPU RAM, set N_BLOCKS = 128
+#   Docs: https://github.com/tromp/cuckoo/blob/master/GPU.md
```

Comment:
Just came across this.. could you add another line here explicitly stating if you have 6GB or less (or you have 6GB and the GPU is also driving the display,) to lower N_BLOCKS?

Reply:
After cuckoo merge [talked about in Lobby](https://gitter.im/grin_community/Lobby?at=5abcac182b9dfdbc3a344ec7) do you know what are the defaults for N_TRIMS and N_BLOCKS? Or I can try to figure it out :)

The commented out lines in grin.toml are:
````
#N_TRIMS = 240
#N_BLOCKS = 128
````
These should probably show the default, then any changes to the default values would be explained separately.

And maybe a line
`# Mining and driving your computer display on the same card? Then maybe you'll need to lower N_TRIMS`

Reply:
N_TRIMS should be adjusted to 256.
N_BLOCKS should be 128 IF you have 6+GB and have enough to spare for your video needs plus other possible GPU applications you like to run.
Otherwise set it to 64, which is 5% less performant on 1080ti.